static/src/javascripts/projects/common/modules/discussion/comment-box.spec.js

```diff
@@ -15,7 +15,7 @@ jest.mock('common/modules/identity/api', () => ({
         id: 1,
         accountCreatedDate: new Date(1392719401338),
     }),
-    getUserFromApi: jest.fn().mockReturnValue(
+    getUserFromApiOrOkta: jest.fn().mockReturnValue(
         Promise.resolve({
```

Comment:
I learned about `mockResolvedValue` the other day! [^1]

I think you can't directly apply this suggestion because there will be extra brackets at the end of the function, but it shows the idea
```suggestion
    getUserFromApiOrOkta: jest.fn().mockResolvedValue({
```

[^1]: https://jestjs.io/docs/mock-function-api#mockfnmockresolvedvaluevalue

Reply:
Nice suggestion!